feat(python): Updates Sanic versions supported (#3860)
getsentry/sentry-docs: src/wizard/python/sanic.md

- `0.8`
- `18.12`
- `19.12`
- `20.12`
- Any version of the form `x.12` (LTS versions).

> **We do support all versions of Sanic.** However, SDK Sanic support for versions between LTS releases should be considered Early Adopters. We might not be supporting all the features in those non-LTS versions, since non-LTS versions [have introduced breaking changes in the past](https://github.com/huge-success/sanic/issues/1532) without prior notice, so we cannot support them as they are too fast of a moving target.

Reviewer: Tried to reword for improved clarity. Does this work?

```suggestion
**We do support all versions of Sanic.** However, Sanic SDK versions between LTS releases should be considered Early Adopter. We may not support all the features in these non-LTS versions, since non-LTS versions change quickly and [have introduced breaking changes in the past](https://github.com/huge-success/sanic/issues/1532), without prior notice. 
```

Author: Yeah this is better, just removed the `SDK` after Sanic cause it probably does not make sense

Author: Thanks!

> **We do support all versions of Sanic.** However, SDK Sanic support for versions between LTS releases should be considered while knowing that we might not be supporting all the features in those non-LTS versions, since non-LTS versions [have introduced breaking changes in the past](https://github.com/huge-success/sanic/issues/1532) without prior notice, so we cannot support them as they are too fast of a moving target.

Reviewer 2: ```suggestion
**We do support all versions of Sanic.** However, SDK Sanic support for versions between LTS releases should be considered Early Adopters. Knowing that we might not be supporting all the features in those non-LTS versions, since non-LTS versions [have introduced breaking changes in the past](https://github.com/huge-success/sanic/issues/1532) without prior notice, so we cannot support them as they are too fast of a moving target.
```
